addons/account/wizard/account_move_send.py

```diff
@@ -122,7 +122,8 @@ def _get_default_mail_subject(self, move, mail_template, mail_lang):
     def _get_default_mail_partner_ids(self, move, mail_template, mail_lang):
         partners = self.env['res.partner'].with_company(move.company_id)
         if mail_template.email_to:
-            for mail_data in tools.email_split(mail_template.email_to):
+            email_to = self._get_mail_default_field_value_from_template(mail_template, mail_lang, move, 'email_to')
```

Comment:
Looks correct at first sight but then we also should do it for the `cc` below isn't it ? :thinking:

Reply:
Thanks for the review. :+1: 

Indeed. I've missed that one: :slightly_smiling_face: